src/System.Security.Permissions/src/System/Security/Permissions/MediaPermission.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+namespace System.Security.Permissions
+{
+    public enum MediaPermissionAudio
+    {
+        NoAudio,
+        SiteOfOriginAudio,
+        SafeAudio,
+        AllAudio
+    }
+    public enum MediaPermissionVideo
+    {
+        NoVideo,
+        SiteOfOriginVideo,
+        SafeVideo,
+        AllVideo,
+    }
+    public enum MediaPermissionImage
+    {
+        NoImage,
+        SiteOfOriginImage,
+        SafeImage,
+        AllImage,
+    }
+    [Serializable()]
+    sealed public class MediaPermission : CodeAccessPermission, IUnrestrictedPermission
+    {
+        public MediaPermission() { }
+        public MediaPermission(PermissionState state) { }
+        public MediaPermission(MediaPermissionAudio permissionAudio) { }
+        public MediaPermission(MediaPermissionVideo permissionVideo) { }
+        public MediaPermission(MediaPermissionImage permissionImage) { }
+        public MediaPermission(MediaPermissionAudio permissionAudio,
+                               MediaPermissionVideo permissionVideo,
+                               MediaPermissionImage permissionImage)
+        { }
+        public bool IsUnrestricted() { return true; }
+        public override bool IsSubsetOf(IPermission target) { return true; }
+        public override IPermission Intersect(IPermission target) { return new MediaPermission(); }
+        public override IPermission Union(IPermission target) { return new MediaPermission(); }
+        public override IPermission Copy() { return new MediaPermission(); }
+        public override SecurityElement ToXml() { return default(SecurityElement); }
+        public override void FromXml(SecurityElement securityElement) { }
+        public MediaPermissionAudio Audio { get; }
+        public MediaPermissionVideo Video { get; }
+        public MediaPermissionImage Image { get; }
```

Comment:
What the general policy in `System.Security.Permissions` about properties?  Should they all be returning nulls?  Should we put a return of a new permission object so that, in case someone was drilling down, then they won't null ref?

Reply:
Doh, I see these are enums, should these return the most permissive to fit with IsUnrestricted?

Reply:
Similarly with other enum returns.

Reply:
I believe the pattern throughout is to return default(T).

Reply:
I think we still want to return the most permissive here, just to be in line with the unrestricted returns.  I know people shouldn't be pivoting on these types, but lining up everything to be as permissive as possible should mean everything sort of just works as if it was full trust in every aspect.